README.md

```diff
@@ -43,6 +43,10 @@ and additionally to support more OpenCV modules:
 
 `apt install libopencv-dev clang libclang-dev`
 
+#### Opensuse:
+
+`zypper install opencv-devel clang-devel gcc-c++`
```

Comment:
Is `gcc-c++` really needed? In theory clang itself should be enough as a c++ compiler

Reply:
Yeah, looks like it. It depends on what we assume a typical dev has installed anyways.

I just tried this using a fresh tumbleweed container. I am running the following commands:

1. `podman run --rm -it docker.io/opensuse/tumbleweed:latest`
2. `zypper install git gcc`   <-- I would assume this to be absolute minimum for a dev setup and not worth extra mentioning...
3. `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh; source "$HOME/.cargo/env"`  <-- same here, need rust to do rust, eh?
4. `zypper install opencv-devel clang-devel` <-- project-specific
5. `git clone git clone https://github.com/twistedfall/opencv-rust/; cd opencv-rust`
6. `cargo build`

And the result is:
```bash
(...)
error occurred: Failed to find tool. Is `c++` installed?
```

which can be fixed by 
8. `zypper install gcc-c++`

Reply:
Hmm, it might be something Suse specific then. The build script for the crate requires both `libclang.so` and `clang` binary to be present and `clang` should provide both C and C++ compilers. But all right, let's consider it resolved :)